tests/unit/karma.conf.js

```diff
@@ -163,7 +163,7 @@ module.exports = function (config) {
 
         // Supress console logs triggered in code.
         client: {
-            captureConsole: true
+            captureConsole: false
```

Comment:
This was accidentally set to true in https://github.com/mozilla/bedrock/pull/13430 so changing back here